# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/moz.build
# Commit: 693a7c400778
# Full Hash: 693a7c4007782b5b3276d4dadefd9cb0a7c71dc6
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-17 05:09:10
# Regressor Bug: 1934497
# File Overlap Count: 2
# Description:
#   Bug 1934497 [Wayland] Implement wl_surface abstraction as WaylandSurface r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230847
# ==============================================================================

diff -r 75d095afa9cf -r 693a7c400778 widget/gtk/moz.build
--- a/widget/gtk/moz.build	Mon Dec 16 20:12:13 2024 +0000
+++ b/widget/gtk/moz.build	Mon Dec 16 20:12:13 2024 +0000
@@ -33,7 +33,6 @@
 
 EXPORTS += [
     "MozContainer.h",
-    "MozContainerSurfaceLock.h",
     "nsGTKToolkit.h",
     "nsGtkUtils.h",
     "nsImageToPixbuf.h",
@@ -56,6 +55,7 @@
     "GtkCompositorWidget.h",
     "InProcessGtkCompositorWidget.h",
     "va_drmcommon.h",
+    "WaylandSurfaceLock.h",
     "WindowSurface.h",
     "WindowSurfaceProvider.h",
 ]
@@ -72,7 +72,6 @@
     "IMContextWrapper.cpp",
     "InProcessGtkCompositorWidget.cpp",
     "MozContainer.cpp",
-    "MozContainerSurfaceLock.cpp",
     "MPRISServiceHandler.cpp",
     "NativeKeyBindings.cpp",
     "NativeMenuGtk.cpp",
@@ -94,6 +93,7 @@
     "ScreenHelperGTK.cpp",
     "TaskbarProgress.cpp",
     "WakeLockListener.cpp",
+    "WaylandSurfaceLock.cpp",
     "WidgetStyleCache.cpp",
     "WidgetUtilsGtk.cpp",
     "WindowSurfaceProvider.cpp",
@@ -112,13 +112,14 @@
         "nsClipboardWayland.cpp",
         "nsWaylandDisplay.cpp",
         "WaylandBuffer.cpp",
+        "WaylandSurface.cpp",
         "WindowSurfaceWaylandMultiBuffer.cpp",
     ]
     EXPORTS.mozilla.widget += [
-        "MozContainerSurfaceLock.h",
         "MozContainerWayland.h",
         "nsWaylandDisplay.h",
         "WaylandBuffer.h",
+        "WaylandSurface.h",
     ]
 
 if CONFIG["MOZ_X11"]:
